contrib/auctex/packages.el

```diff
@@ -54,10 +55,36 @@
         "mhd" 'TeX-doc ;; TeX-doc is a very slow function
         )
 
+      (setq-default TeX-command-default auctex-build-command)
+
       (setq-default TeX-auto-save t)
       (setq-default TeX-parse-self t)
-      (setq-default TeX-PDF-mode t))))
+      (setq-default TeX-syntactic-comment t)
+      (setq-default TeX-PDF-mode t)
+
+      (when auctex-electric-sub-and-superscript
+        (setq-default TeX-electric-sub-and-superscript t))
+      (when auctex-electric-escape
+        (setq-default TeX-electric-escape t))
+      (setq-default TeX-electric-macro)
+
+      ;; Synctex support
+      (setq-default TeX-source-correlate-mode t)
+      (setq-default TeX-source-correlate-start-server nil)
+
+      ;; Setup reftex style (RefTeX is supported through extension)
+      (setq-default reftex-use-fonts t)
 
+      ;; Don't insert line-break at inline math
+      (setq-default LaTeX-fill-break-at-separators '()))))
+
+(when (string= auctex-build-command "LatexMk")
```

Comment:
So I'm the one to not understand it correctly. I thought that `auctex-build-command` was a layer variable you created and it was used just to enable the package `auctex-latexmk`.
When there is multiple choice for the layer variable we usually use symbols instead of strings.